Admin/Get-EASMailboxLogs.ps1

```diff
@@ -2,13 +2,39 @@
 # Licensed under the MIT License.
 
 # Powershell script to enable and collect EAS mailbox logs.
+[CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Medium')]
 param(
     [Parameter(Mandatory = $true)] [string[]] $Mailbox = "",
     [ValidateScript({ Test-Path $_ })]
     [Parameter(Mandatory = $true)] [string] $OutputPath = (Get-Location).Path,
-    [Parameter(Mandatory = $false)] [int] $Interval = 30
+    [Parameter(Mandatory = $false)] [int] $Interval = 30,
+    [Parameter(Mandatory = $false)] [nullable[bool]] $EnableMailboxLoggingVerboseMode = $null
 )
 
+#parse $EnableMailboxLoggingVerboseMode and convert it boolean value(s)
+switch ($EnableMailboxLoggingVerboseMode) {
+    $true { $EnableVerboseLogging = "true" }
+    $false { $EnableVerboseLogging = "false" }
+    default { $EnableVerboseLogging = $null }
+}
+
+#Override EnableMailboxLoggingVerboseMode key's value with EnableVerboseLogging
+if ($null -ne $EnableVerboseLogging) {
+    if ($PSCmdlet.ShouldProcess("Set EnableMailboxLoggingVerboseMode attribute to $EnableVerboseLogging in $env:ExchangeInstallPath" + "ClientAccess\Sync\web.config", 'TARGET', 'OPERATION')) {
+        try {
+            [xml]$web = Get-Content $env:ExchangeInstallPath"ClientAccess\Sync\web.config"
+        } catch {
+            Write-Error ("Failed to read $env:ExchangeInstallPath" + "ClientAccess\Sync\web.config")
+            exit
+        }
+        Copy-Item $env:ExchangeInstallPath"ClientAccess\Sync\web.config" -Destination $env:ExchangeInstallPath"ClientAccess\Sync\web.config.bak"
+        $web.SelectSingleNode('//add[@key="EnableMailboxLoggingVerboseMode"]').Value = $EnableVerboseLogging
+        $web.Save($env:ExchangeInstallPath + "ClientAccess\Sync\web.config")
```

Comment:
Include all of this in the `try` `catch` block

Reply:
Separate block for each call? To have clear error explanation?